test/inject.test.ts

```diff
@@ -1,4 +1,4 @@
-import {Inject} from "../core";
+import {Inject} from "../decorators/inject";
```

Comment:
I think we should keep importing from core.

Reply:
Why is that important?